internal-services/catalog/iib-add-fbc-fragment-to-index-image-task.yaml

```diff
@@ -131,13 +135,19 @@ spec:
             mustSignIndexImage="true"
             mustPublishIndexImage="true"
         else
-            echo "Fragment has \`fbc_opt_in==${fbcOptIn}\`"
-            # in case of more than one image all should have the same fbc_opt_in value
-            if [ $(wc -w <<< ${fbcOptIn[@]}) -ne 1 ]; then
-                fbcOptIn="false"
-                mustSignIndexImage="false"
-                mustPublishIndexImage="false"
-            fi
+            if [ "$(params.stagedIndex)" == "true" ]; then
```

Comment:
Please give it a thought as well, but I think we can remove two levels of nesting like this:
```sh
if [ "$(params.hotfix)" == "true" ]
...
elif [ "$(params.stagedIndex)" == "true" ]
...
elif [ $(wc -w <<< ${fbcOptIn[@]}) -ne 1 ]
...
fi
```
And then perhaps add the echo after all of that:
```sh
echo "Fragment has \`fbc_opt_in==${fbcOptIn}\`"
```